d2plugin/plugin_elk.go

```diff
@@ -85,8 +85,11 @@ func (p elkPlugin) Info(ctx context.Context) (*PluginInfo, error) {
 		f.AddToOpts(opts)
 	}
 	return &PluginInfo{
-		Name:      "elk",
-		Type:      "bundled",
+		Name: "elk",
+		Type: "bundled",
+		Features: map[PluginFeature]struct{}{
```

Comment:
This should be an array not a map, even if that makes checking for a feature a little harder.

Reply:
why? the ordering does not matter.

Reply:
i guess if they print help and the order differs each time b/c of map iteration not being deterministic, that's bad.